packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -176,9 +176,12 @@ export class OdspDocumentService implements IDocumentService {
         // batch size, please see issue #5211 for data around batch sizing
         const batchSize = this.hostPolicy.opsBatchSize ?? 5000;
         const concurrency = this.hostPolicy.concurrentOpsBatches ?? 1;
-
+        const messages: ISequencedDocumentMessage[] = [];
```

Comment:
I'd rathe push that conversion up the stack (into this.storageManager, better - up to the point where we get specific wire format (i.e. IOdspSnapshot) and parse it. That way it's so much easier to find it and change if format is going to change, without impact to downstream code.

Reply:
Also can we avoid cast to any? I.e. have some more formal check in some form?